packages/view-types/feature-list/src/selectable-table/constants.js

```diff
@@ -0,0 +1,9 @@
+export const FEATURELIST_SORT_OPTIONS = [
+  'alphabetical',
+  'original',
+];
+
+export const FEATURELIST_SORT_KEYS = [
+  'Gene ID',
+  'Alternate ID',
```

Comment:
```suggestion
  'featureIndex',
  'featureLabels',
```

To generalize to non-gene cases, can we use these as the internal+prop values (while still using `${featureType} ID` and `Alternate ID` as the user-facing values)?

 (Sorry, I know this change will require changes in a few more places where 'Gene ID' and 'Alternate ID' had been hard-coded)

Reply:
no worries, it is a good point!